ui/src/utils/prop-types.js

```diff
@@ -254,16 +254,15 @@ export const awsProvisionerWorkerType = shape({
 });
 
 export const WorkerManagerWorkerPoolSummary = shape({
-  workerPool: string,
-  pendingCapacity: number,
-  runningCapacity: number,
+  workerPoolId: string,
+  owner: string,
   pendingTasks: number,
   lastActive: date,
   lastResolved: date,
   failed: number,
   exception: number,
   unscheduled: number,
-  provider: string,
+  providerId: string,
 });
```

Comment:
I think the properties in here should reflect the values in https://github.com/taskcluster/taskcluster/blob/master/services/worker-manager/schemas/v1/worker-pool-full.yml.

Reply:
Can you point out the specific problem you see here?

Reply:
I think things like `failed` and `exception` are not part of the worker-pool schema. It's also missing a couple of fields like `config` and `lastModified`.

Reply:
I am not clear on the connection between the `prop-types` and the schema. However, I am getting rid of those properties in this PR (I listed the intended changes in the PR description), and I forgot to remove them from here. So I removed them.

Reply:
The input which we are feeding `WorkerManagerWorkerPoolsTable` comes from the `WorkerManagerWorkerPoolSummaries` graphql query. Right now we are only setting the ones that we are requiring in that specific view but technically we could have another view that uses the same query but fetches a different set of fields. Because of that, we should list all of the fields but have them as optional (i.e., they don't have the `.isRequired` specified).

```jsx
export const WorkerManagerWorkerPoolSummary = shape({
  workerPoolId: string,
  providerId: string,
  description: string,
  created: date,
  lastModified: date,
  config: object,
  owner: string,
  emailOnError: bool,
});
```

Reply:
oh I see. I'll make it match the schema - the only question is `pendingTasks` field ,as it's not in the schema and come from a different service.... I see in the code, the prop-type for aws provisioner does have that field - I'll do the same